### DISCUSSION TITLE
how change TLS to 1.2

### DISCUSSION THREAD

**Discussion by lahirubro123** [FranzKafkaYu/x-ui#251] [Q&A]
"minVersion": "1.2",
  "maxVersion": "1.3",

how add those flags to x-ui

![image](https://user-images.githubusercontent.com/72745298/216339273-380cf227-4385-4ea9-838d-85f33f50cdfe.png)

**Comment by FranzKafkaYu** [comment]
@lahirubro123 it seems only my version supports this feature for now.

**Reply by lahirubro123** [reply]
can you add GB flow limit and expire limit / ip limit for each user separately ?
